coalib/parsing/DefaultArgParser.py

```diff
@@ -155,6 +155,10 @@ def default_arg_parser(formatter_class=None):
         '--no-autoapply-warn', const=True, action='store_const',
         help='turn off warning about patches not being auto applicable')
 
+    config_group.add_argument('--init', nargs=2, metavar='FILE',
```

Comment:
A better integration mechanism would be to provide a way to bootstrap a `.coafile` instead.

For example,
```
coala --generate-config py,cpp,html 
```

And this utility should just download corresponding style files from a repo, collate them and save as `.coafile`.

Reply:
`moban` integration at a later point can add a whole lot more features.

Reply:
https://github.com/PrajwalM2212/coala-styles/issues/1#issuecomment-520203605

Reply:
@ksdme Done.
Users can now run `coala -g python pep8 -g cpp google` and this will bootstrap the .coafile with configurations for pep8 and cpp google style guides.